fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonTypeDefinitions.java

```diff
@@ -175,34 +175,34 @@ public static String pythonType (Type type) {
 	 */
 	public static String pythonTypeHL (Type type) {
 		if (type instanceof VoidType)				return "'None'";
-		if (type instanceof BooleanType)			return "'bool'";
```

Comment:
Why were all these changed?
The definitions for `pythonTypeAttrHL` were not updated.  Do they need to be?

Reply:
The data type generated in 'dtype' are in short forms like 'str', 'uint16' or '('str',)'. So changed the 'dtype' as per the user selection for better understanding. For example, 'str' to 'DevString' and 'int16' to 'DevShort', etc.

Agree with you regarding definitions update for other methods. I will updates for pythonTypeAttrHL as well as for pythonPropTypeHL. In this request, I have only focused on command generation.

Reply:
Updated data type definitions for method pythonTypeAttrHL and pythonPropTypeHL. For method pythonPropTypeHL, data type definitions are referred from the 'CppTypeDefinitions.java' file. 
The earlier definitions generated are a bit similar for different data types. For example, for DevString, it is returned as 'str' and for DevVarStringArray, it is returned as '('str',)'. For documentation and understanding perspective, have changed as per the data type definition values.
Link to commit on fork:
https://github.com/ska-telescope/pogo/commit/9607f30c42cc62f0e60007950774cf54153d2091

Reply:
It was changed for better documentation. In command doc string, data type of argument is also generated. So in read the docs documentation for better understanding of developer I had made that change. Also in Python code it is mapped to Tango types. If it might cause issue, then it can be changed similar to earlier version. Please confirm.

Reply:
OK, it sounds like you have a strong preference to the TANGO types in the Python code, so we can leave it that way.  The functionality of the code will be identical.

Reply:
Okay (y)